api_tests/tests/reasonable_adjustments/test_happy_cases.py

```diff
@@ -9,15 +9,24 @@
 import uuid
 import base64
 
+
 @pytest.mark.usefixtures("setup")
 class TestHappyCasesSuite:
     """ A test suite to verify all the happy path oauth endpoints """
 
     @pytest.mark.happy_path
     @pytest.mark.integration
     @pytest.mark.smoke
-    @pytest.mark.usefixtures('get_token_internal_dev')
-    def test_consent_get(self):
+    @pytest.mark.sandbox
+    @pytest.mark.usefixtures('get_token_for_parameterized_headers')
+    @pytest.mark.parametrize('headers', [
```

Comment:
`parametrize` is used when we want to run the same test multiple times, each time with different parameter. This is not the case with this (and other) tests. We only have one possible set of headers for each test.
I think leaving headers inside the body would be better.

Reply:
As discussed on our call, I'll remove the logic of not passing in the header whatsoever and keep it that for sandbox we will pass in `Authorization: Bearer` (without a token) as this header is ignored by the sandbox anyway